packages/MarkdownEditor-Core.package/MarkdownBlockTextStyler.class/instance/buildAttributeRunsFrom..st

```diff
@@ -1,6 +1,8 @@
-parsing
+styling
 buildAttributeRunsFrom: aCollectionOfMarkdownBlocks
-	
-	^ (aCollectionOfMarkdownBlocks 
-		inject: Text new 
-		into: [:text :block | text append: block asText]) runs
+
+	| text |
+	text := Text new.
+	aCollectionOfMarkdownBlocks do: [:block |
+		text append: block asText].
```

Comment:
I liked `inject:into:` but I think a simple `do:` shows that there is no magic happening here...

Reply:
I personally don't see a improvement in this.

Reply:
So, is it a request to change it back or are you neutral which version we should take?

Reply:
Yes it is. I think the `inject` statement fits the solution a little more.

Reply:
I tried to make a version which uses `inject:into:` and shows explicitly that we take the runs of a text, but it always resulted in overhead.

```smalltalk
| text |
text := aCollectionOfMarkdownBlocks inject: Text new into: [:accumulatedText :block | accumulatedText append: block asText].
^ text runs
```

I'm also wary because we don't return a new object by using `inject:into:`, but modify one text object. In the normal cases, you always make a new object (like in `inject: 0 into: [:acc :number | acc + number]`), but I dunno whether you would do it in our case...

Reply:
I don't see the problem with the `inject:into:` code in your last comment. Obviously it needs line breaks but there is no overhead that I can see.